index.md

```diff
@@ -1,51 +1,61 @@
 
-Additional topics may be created as desired. But, in general, Kafka performs better with a small number of large topics. For each (upstream) input topic, there is an associated (downstream) notification topic for [batch status notifications](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json). 
+As needed, you can create additional topics. Generally, Kafka performs better with a small number of large topics. For each (upstream) input topic, there is an associated (downstream) notification topic for batch status notifications. To review the relevant JSON for notifications, see the [API specification on GitHub](https://github.com/Alvearie/hri-api-spec/tree/master/notifications/batchNotification.json). 
 
 ### Batches
-Health Record Datasets often have organizational requirements to be processed together "as a set" (partially or in their entirety) when moving the data into a new cloud instance. Hence, HRI has been built with support to process a dataset as a _Batch_. See [Batch](glossary.md#batch) for a detailed definition.  
-
-How much data goes in a batch is really up to the Data Integrator. The Management API provides support for starting, completing, terminating, and searching for batches. Any change to a batch results in a message being written to the associated notification topic in Kafka. 
-
-### Data Format
-HRI does not impose any requirements on the format of the Health Data records written to Kafka. There is a separate effort to define a common FHIR model for PHI data. 
-
-However, The HRI **does require** the `batch Id` to be in the record header. Data Integrators may include any number of additional custom header values that they wish to pass onto data consumers. An example of a custom header value might be something like `originating_producer_id`, an originating data producer (or org) id value that may need to be communicated to the data consumers. 
-
-## Additional Reading
-- [Processing Flows](processflow.md)
-- [API specification](apispec.md)
-- [Multi-tenancy](multitenancy.md)
-- [Dependency Config](config-setup.md)
-- [Administration](admin.md)
-- Estimating Cost
-- [Monitoring & Logging](monitorlog.md)
-- [Troubleshooting](troubleshooting.md)
-- [Glossary](glossary.md)
-- [Roadmap](roadmap.md)
-
-## Questions
-Please contact our Maintainers for further questions: 
-  * David N. Perkins, Team Lead: david.n.perkins@ibm.com
-  * Aram S. Openden, Maintainer: aram.openden1@ibm.com
+
+When health record datasets are moved into a new cloud instance, the data
+often has organizational requirements to be processed together as a dataset called a [**batch**](glossary.md#batch). Processing can occur with either a partial batch or an entire batch. For these reasons, Health Record Ingestion has been built with to support batch dataset processing. 
```

Comment:
Remove the link break after 'the data'.  We need to be clear that a 'batch' is an HRI term and not one generally accepted, which this change makes less clear. The 'organization' requirements are also not related to the cloud, so I would remove the introductory phrase 'When health record datasets are moved into a new cloud instance'.